### PR TITLE
Remote cuDF Merge

### PR DESCRIPTION
**Pull request by quasiben**
PR adds a remote version of the cuDF merge benchmark.  This is intended to be used when a cluster is launched separately from the client.  I've been using this can be used to help stress test new clusters

### CODE REVIEW

**Comment by quasiben** [comment]
rerun tests

**Comment by quasiben** [comment]
rerun tests

File: dask_cuda/explicit_comms/dataframe_merge.py [hunk 1]
```diff
@@ -93,7 +93,7 @@ def partition_by_hash(df, columns, n_chunks):
     # Hashing `columns` in `df` and assing it to the "_partitions" column
     df["_partitions"] = partitioning_index(df[columns], n_chunks)
     # Split `df` based on the hash values in the "_partitions" column
-    ret = shuffle_group(df, "_partitions", 0, n_chunks, n_chunks)
+    ret = shuffle_group(df, "_partitions", 0, n_chunks, n_chunks, ignore_index=True)
```
Comment (quasiben) [inline dask_cuda/explicit_comms/dataframe_merge.py]: @rjzamora I needed to added the `ignore_index`.  I want to double check with you that this should be set to `True`

Comment (madsbk) [reply dask_cuda/explicit_comms/dataframe_merge.py]: Fixed the missing `ignore_index` in https://github.com/rapidsai/dask-cuda/pull/253

**Comment by quasiben** [comment]
I would vote for shared code (utils.py) mostly because the cluster.scheduler calls change between remote/local but I'm open to suggestion.   Do you have a preference ?

**Comment by madsbk** [comment]
> I would vote for shared code (utils.py) mostly because the cluster.scheduler calls change between remote/local but I'm open to suggestion. Do you have a preference ?

Agree, having a utils.py within benchmarks for shared code is a good approach.

**Comment by quasiben** [comment]
closing in favor of https://github.com/rapidsai/dask-cuda/pull/293

**Comment by pentschev** [comment]
@quasiben sorry I was completely unaware or had forgotten about this. Is there still anything valuable here that you think we could move into #293 ?

**Comment by quasiben** [comment]
No, your implementation is a lot better and I let this drop

**Pull request closed without merging**